spec/requests/api/v1/collections_spec.rb

```diff
@@ -46,124 +55,167 @@
     context 'a required parameter is empty/blank' do
       before do
         invalid_params = {}
-        post "/api/v1/collections", params: invalid_params, headers: valid_session
+        post '/api/v1/collections', params: invalid_params, headers: valid_session
       end
 
       it 'returns failure message as JSON' do
         expect(response.status).to eq(400)
-        expect(JSON.parse(response.body)).to match(hash_including('status' => 400, "developer_message" => "handle is missing, handle is empty, token is missing, token is empty"))
+        expect(JSON.parse(response.body)).to match(
+          hash_including('status' => 400,
+                         'developer_message' => 'handle is missing, handle is empty, token is missing, token is empty')
+        )
       end
```

Comment:
Can we change this to not use hash rockets on areas you cleaned up?
hash_including('status': 400,
                         'developer_message': 'handle is missing, handle is empty, token is missing, token is empty')
        )

Reply:
@peggles2 , I was trying to avoid the extra refactoring that would require, as the JSON being tested uses string keys. That said...I suppose there's no time like the present, so I'll do some extra tweaking.